transform/instance.go

```diff
@@ -390,8 +392,8 @@ func (ot *objectTransformer) objectTransformXML(in interface{}, modifier pathMod
 			if len(v) > 0 {
 				in = v[0]
 			}
-		default:
-			return nil, errors.New("non xml node returned from object transform")
+		case nil:
+			foundTransformObjectNode = false
```

Comment:
I don't see how using this bool is effectively any different than just `return ot.defaultValue, nil`